script/test.mjs

```diff
@@ -1,43 +1,23 @@
 import { spawn } from 'child_process'
-import { glob } from 'glob'
```

Comment:
So turns out I was depending on a sub-dependency of rimraf here so when I removed it `glob` disappeared as well but that lead to a nice cleanup in this script as well, utilizing the recursive option of readdir

Reply:
Nice cleanup indeed 🥹